src/ol_infrastructure/substructure/vault/setup/pulumi_policy.hcl

```diff
@@ -0,0 +1,6 @@
+path "*" {
+  capabilities = ["create", "read", "update", "delete", "list"]
+}
+path "sys/*" {
+  capabilities = ["create", "read", "update", "delete", "list"]
+}
```

Comment:
Any way to be more restrictive with those permissions?

Reply:
It's certainly possible, but then we'll likely end up doing whack-a-mole whenever we're trying to process things in Vault